framework/doc/content/source/kernels/ArrayDiffusion.md

```diff
@@ -26,7 +26,7 @@ We can rearrange the fully expanded [eq:weak-array-diffusion] into
 \begin{equation}
 (\nabla \vec{u}^\ast, \mathbf{D} \nabla \vec{u}) = \sum_{e} \sum_{i=1}^{N_{\text{dof}}} \sum_{\text{qp}=1}^{N_{qp}} (|J|w)_{\text{qp}} \vec{w}_p\vec{u}_i^\ast \underline{\mathbf{D}_{\text{qp}} \nabla \vec{u}_{\text{qp}} \cdot \nabla b_{i,\text{qp}}},
 \end{equation}
-where the underlined term is the array provided by [ArrayDiffusion::computeQpResidual](ArrayDiffusion.C).
+where the underlined term is the array provided by [ArrayDiffusion::computeQpResidual](/ArrayDiffusion.C).
```

Comment:
Just curious about what the purpose of the leading slash is. Is that to disambiguate ADArrayDiffusion.C and ArrayDiffusion.C?